src/mrpro/operators/models/_MOLLI.py

```diff
@@ -37,20 +38,23 @@ def forward(self, a: torch.Tensor, b: torch.Tensor, t1: torch.Tensor) -> tuple[t
         Parameters
         ----------
         a
-            Parameter a in MOLLI signal model
+            parameter a in MOLLI signal model
+            with shape (... other, coils, z, y, x)
         b
-            Parameter b in MOLLI signal model
+            parameter b in MOLLI signal model
+            with shape (... other, coils, z, y, x)
         t1
-            longitudinal relaxation time T1 [s]
+            longitudinal relaxation time T1
+            with shape (... other, coils, z, y, x)
 
         Returns
         -------
-            signal with dimensions ((... inv_times), coils, z, y, x)
+            signal
+            with shape (time ... other, coils, z, y, x)
         """
-        t1 = torch.where(t1 == 0, 1e-10, t1)
-        a = torch.where(a == 0 | torch.equal((b / a), torch.ones_like(a)), a + 1e-10, a)
-        ti = self.ti[(...,) + (None,) * (a.ndim)]
-        t1_star = t1 / ((b / a) - 1)
-        y = a - b * torch.exp(-(ti / (t1_star)))
-        res = rearrange(y, 't ... c z y x -> (... t) c z y x')
-        return (res,)
+        delta_ndim = a.ndim - (self.ti.ndim - 1)  # -1 for time
```

Comment:
what if a and b have different dimensions...?

Reply:
This is not supported, correct? 
..and not worth supporting I would say..

Reply:
> ..and not worth supporting I would say..

Agreed.